deploy/Jenkinsfile

```diff
@@ -0,0 +1,227 @@
+pipeline {
+    agent {
```

Comment:
Could you move these files to test directory rather than new deploy?

Reply:
I will create a folder in `/tests` called `jenkins` and move all scripts used in the pipeline there